pkg/render/node.go

```diff
@@ -746,6 +753,12 @@ func (c *nodeComponent) nodeVolumes() []v1.Volume {
 	return volumes
 }
 
+func (c *nodeComponent) bpfDataplane() bool {
```

Comment:
For what this function does, I would prefer for the function name to be more of a question, something like `bpfDataplaneEnabled` or `isBpfDataplaneEnabled`. WDYT? I feel like it could improve readability slightly but not excessively so if you don't think that makes sense then feel free to leave it as it is.